### PR TITLE
Added meta tags and favicon

### PR DESCRIPTION
**Pull request by IndhujaMuthusamy**
Changes:
- Added meta tags for og
- Added meta tags for twitter
- Added images for favicon and metatag image

Testing:
-Tested favicon

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,7 +4,20 @@
...
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta property="og:image" content="/jovian_meta.png">
+    <meta name="twitter:title" content="Jovian Careers">
+    <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="twitter:image" content="/jovian_meta.png">
+    <title>Jovian Careers</title>
+    <link rel="icon" href="/jovian_favicon.png" type="image/x-icon">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (SurendarKailasam) [inline src/index.html:20]: Improve the title text

**Review by SurendarKailasam — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -4,7 +4,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My First Web Page</title>
+    <meta name="description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Jovian">
+    <meta name="robots" content="index, follow">
```
Comment (SurendarKailasam) [inline src/index.html:10]: test

**Review by SurendarKailasam — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (IndhujaMuthusamy) [reply src/index.html:20]: improved

File: src/index.html (hunk 2 above)
Comment (IndhujaMuthusamy) [reply src/index.html:10]: ok

**Pull request merged**